### PR TITLE
PPNA-4785 Add basic python linters

### PR DESCRIPTION
**Pull request by pulina**
Use example https://github.com/PiwikPRO/MMDBSequentialReader/runs/6856935090?check_suite_focus=true

### CODE REVIEW

File: python/lint/action.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+runs:
+  using: "composite"
+  steps:
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+    - name: Run isort linter
+      if: ${{ inputs.use-isort == 'true' }}
+      uses: jamescurtin/isort-action@master
```
Comment (mrfijas) [inline python/lint/action.yaml:29]: I don't think we should use 3rd party actions without pinning it to commit hash and reviewing its codebase..

and speaking of 3rd party - if we use it anyway, isn't there some all-in-one proven-in-action things? like some https://github.com/ricardochaves/python-lint that's wrapping black, pylint, isort, pycodestyle, flake8, mypy?

File: python/lint/flake8.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 100
+max-complexity = 10
+exclude = *build*,*artifacts*
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,D203,D213,W504
```
Comment (mrfijas) [inline python/lint/flake8.ini:5]: not sure about that list - it's some arbitrary list of Reporting's ancestors, right?

**Review by mrfijas — APPROVED** [review]
we could use it as a starting point for discussion about linters used in multiple repositories of multiple teams

File: python/lint/flake8.ini (hunk 2 above)
Comment (pulina) [reply python/lint/flake8.ini:5]: Right. But this list is reasonable.

**Review by MarcinMaslany — APPROVED** [review]

**Pull request merged**